index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-cycle */
+import { bookListHandler, addSectionHandler, contactInfoHandler } from './modules/singlePage.js';
+import { DateTime } from './modules/luxon.js';
+import Book from './modules/books.js';
+import displayToPage from './modules/desplayToPage.js';
+
+const linForBookList = document.querySelector('.for_list_link');
+const linkForAddSection = document.querySelector('.for_addSection_link');
+const linkForContInfo = document.querySelector('.for_contact_link');
+
+const addButton = document.querySelector('#add');
+const titleInput = document.querySelector('#title');
+const autherInput = document.querySelector('#author');
+/* eslint-disable import/no-mutable-exports,import/prefer-default-export */
+export let booksData = [];
+/* eslint-disable import/no-mutable-exports,import/prefer-default-export */
```

Comment:
- [ ] Let's try not to disable Linter checks in our codebase as they help us write better and more professional code. Disabling Linter checks defies the purpose of having it and hence does not help you write professional code. Kindly fix the Linter errors 👍🏽